FrameworkSystem/Client/AuthManagerClient.py

```diff
@@ -0,0 +1,46 @@
+""" DIRAC AuthManager Client class encapsulates the methods exposed
+    by the AuthManager service.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Core.Base.Client import Client, createClient
+from DIRAC.Core.Utilities import DIRACSingleton
+from DIRAC.FrameworkSystem.Client.AuthManagerData import gAuthManagerData
+
+__RCSID__ = "$Id$"
+
+
+@createClient('Framework/AuthManager')
+class AuthManagerClient(Client):
+  """ Authentication manager
+  """
+  __metaclass__ = DIRACSingleton.DIRACSingleton
```

Comment:
This is not python 3 compatible.

Reply:
To solve this issue, I used package six:
```
@six.add_metaclass(DIRACSingleton.DIRACSingleton)
class AuthManagerClient(Client):
```